src/colmap/sfm/incremental_triangulator.cc

```diff
@@ -302,7 +302,9 @@ size_t IncrementalTriangulator::MergeAllTracks(const Options& options) {
   return num_merged;
 }
 
-size_t IncrementalTriangulator::Retriangulate(const Options& options) {
+size_t IncrementalTriangulator::Retriangulate(
+  const Options& options,
+  const std::unordered_set<image_t>& ignore_image_ids) {
```

Comment:
Let's use a pointer and default to a nullptr instead of checking for ".empty()".

```suggestion
  const std::unordered_set<image_t>* ignore_image_ids) {
```